flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/src/main/java/BatchJob.java

```diff
@@ -32,28 +32,29 @@
  * 		mvn clean package
  * in the projects root directory.
  * You will find the jar in
- * 		target/flink-quickstart-0.1-SNAPSHOT-Sample.jar
- *
+ * 		target/flink-quickstart-0.1-SNAPSHOT.jar
+ * From the cli you can then run
+ * 		./bin/flink run -c  &lt;BatchJob&gt; &lt;quickstart jar&gt;
```

Comment:
1 Excessive space after '-c'.
Also, maybe this would be more specific: `./bin/flink run -c ${package}.BatchJob target/flink-quickstart-0.1-SNAPSHOT.jar`